Lib/idlelib/idle_test/test_help_about.py

```diff
@@ -170,5 +176,37 @@ def test_file_display(self):
                 self.assertEqual(self.view.called, True)
 
 
+class BitnessTest(unittest.TestCase):
+    "Test helper function for bitness of system."
+
+    def test_darwin(self):
+        help_about.sys = mock.MagicMock()
+        platform = mock.PropertyMock(return_value='darwin')
+        size = mock.PropertyMock(return_value=2**32)
+        type(help_about.sys).platform = platform
+        type(help_about.sys).maxsize = size
+        help_about.architecture = mock.Mock(return_value=('64bit', 'ELF'))
+        self.assertEqual(help_about._bitness(), '32')
+        platform.assert_called_with()
+        size.assert_called_with()
+
+        size = mock.PropertyMock(return_value=2**64)
+        type(help_about.sys).maxsize = size
+        self.assertEqual(help_about._bitness(), '64')
+        platform.assert_called_with()
+        size.assert_called_with()
+
+    def test_not_darwin(self):
+        help_about.sys = mock.MagicMock()
+        platform = mock.PropertyMock(return_value='linux')
+        size = mock.PropertyMock(return_value=2**32)
+        type(help_about.sys).platform = platform
+        type(help_about.sys).maxsize = size
+        help_about.architecture = mock.Mock(return_value=('64bit', 'ELF'))
+        self.assertEqual(help_about._bitness(), '64')
+        platform.assert_called_with()
+        size.assert_not_called()
```

Comment:
If build bits were a real function, I would want to to try to hit all branches.  And if we otherwise get to to 99'% coverage, I might want to do so, but it would be artificial.  On Macs, the value of sys.architecture seems undefined, which is why it is not used.

Read something about 'unit testing', such as parts of https://en.wikipedia.org/wiki/Unit_testing and 'test-driven development', such as the first half of https://en.wikipedia.org/wiki/Test-driven_development.  There is clearly overlap.  Keep in mind that both topics are subjects of programming religion.